FirebaseMLModelDownloader/Sources/ModelDownloader.swift

```diff
@@ -238,7 +238,7 @@ public class ModelDownloader {
       var customModels = Set<CustomModel>()
       for path in modelPaths {
```

Comment:
Variable names `path` and `modelPaths` look a bit confusing as they are URLs in fact.

Reply:
Good point. I updated modelPath to be called modelURL so it's clearer.